DSharpPlus/Net/Rest/DiscordApiClient.cs

```diff
@@ -876,7 +874,8 @@ internal async Task<DiscordMessage> CreateMessageAsync(ulong channel_id, Discord
                 Content = builder.Content,
                 IsTTS = builder.IsTTS,
                 HasEmbed = builder.Embed != null,
-                Embed = builder.Embed
+                Embed = builder.Embed,
+                Components = builder.Components.ToList().AsReadOnly() // if you change the builder and it blows up, it's on you ~Velvet //
```

Comment:
`ToList().AsReadOnly()` is completely unnecessary here, it's already the right type. Also, remove the comment, as it's no longer relevant.